rsocket/transports/tcp/TcpDuplexConnection.cpp

```diff
@@ -210,7 +211,7 @@ class TcpInputSubscription : public Subscription {
 
   void request(int64_t n) noexcept override {
     DCHECK(tcpReaderWriter_);
-    DCHECK(n == kMaxRequestN)
+    DCHECK_EQ(n, std::numeric_limits<int64_t>::max())
```

Comment:
The short answer is no, its not a problem.
We have 2 max values to use. The yarpl library is using int64_t max value and the rsocket protocol is using int32_t max value.
In this case we are using yarpl interfaces and the value is never sent over the network via rsocket.